src/custom-element.js

```diff
@@ -315,7 +315,7 @@ export function applyLayout_(element) {
 
   // Apply UI.
   element.classList.add(getLayoutClass(layout));
-  if (isLayoutSizeDefined(layout)) {
+  if (isLayoutSizeDefined(layout) && !element.isAwaitingSize_()) {
```

Comment:
Why not set this class right away? I'd like to scope elements right away to avoid any kind of nuances with build callbacks.